src/Resources/contao/dca/tl_settings.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/*
+ * News Categories bundle for Contao Open Source CMS.
+ *
+ * @copyright  Copyright (c) 2020, Codefog
+ * @author     Codefog <https://codefog.pl>
+ * @license    MIT
+ */
+
+use Contao\CoreBundle\DataContainer\PaletteManipulator;
+
+$GLOBALS['TL_DCA']['tl_settings']['fields']['news_categorySlugSetting'] = [
+    'inputType' => 'select',
+    'options_callback' => static function () {
```

Comment:
I would prefer to have a reference to an event listener here, instead of having the anonymous function.

Reply:
Should I create a `SettingsListener` class for this, with an `onSlugSettingOptions` method?

Reply:
That would be superb, thank you 🙂